### PR TITLE
Error Messages specifies individual errors

### PR DESCRIPTION
**Pull request by brian16600**
Update Error messages for the following command and their specifics:
1. addc
- displays missing compulsory parameters + correct format

2. addt 
- displays missing compulsory parameters
- displays date error message for wrong format
- Recurrence format is invalid
- Recurrence index is invalid

3. assignc
- displays missing parameter

4. unassignc
- displays missing parameter

5. deletec
- displays invalid contact index

6. deletet
- displays invalid task index

7. editc
- displays invalid contact index

8. editt
- displays no parameters entered
- displays invalid task index
- displays invalid datetime

9. findc
- displays no keywords specified

10. gen
- displays invalid task index

11. mark
- displays invalid task index

12. unmark
- displays invalid task index

13. view
- displays invalid task index

### CODE REVIEW

**Comment by brian16600** [comment]
Updated relevant test cases as well

File: docs/UserGuide.md [hunk 1]
```diff
@@ -111,12 +111,12 @@ Format: `help`
...
 Adds a contact.
 
-Format: `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GIT_USERNAME [t/TAGS]`
+Format: `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GITHUB_USERNAME [t/TAGS]`
 
 Examples:
 
 * `addc n/john p/12345678 e/john@nus.edu.sg u/john123 t/Schoolmate`
-* `addc n/mary p/87654321 e/mary@nus.edu.sg u/maryCS t/Teammate t/Classmate`
+* `addc n/mary p/87654321 e/mary@nus.edu.sg u/maryCS t/Tea![img.png](img.png)Hmmate t/Classmate`
```
Comment (junrong98) [inline docs/UserGuide.md:119]: I think you accidentally edited this?

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java [hunk 2]
```diff
@@ -25,23 +25,25 @@
...
-            + " [" + PREFIX_TAG + "TAG] "
-            + " [" + PREFIX_LINK + "LINK] "
-            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE]\n"
+            + " [" + PREFIX_TAG + "TAG" + "] "
+            + " [" + PREFIX_LINK + "LINK" + "] "
+            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE" + "] \n"
             + "Example: " + "addt" + " "
-            + PREFIX_TASKNAME + "Lecture "
-            + PREFIX_DATETIME + "25-12-2022 1800 "
+            + PREFIX_TASKNAME + "John Doe "
```
Comment (junjunjieOng) [inline src/main/java/seedu/address/logic/commands/AddTaskCommand.java:38]: Task name should keep as "Lecture"

**Review by junrong98 — COMMENTED** [review]
Overall, LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (brian16600) [reply docs/UserGuide.md:119]: Ahh I think i put in the wrong one in the merging, will change it

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java (hunk 2 above)
Comment (brian16600) [reply src/main/java/seedu/address/logic/commands/AddTaskCommand.java]: Ok changed

File: docs/UserGuide.md [hunk 3]
```diff
@@ -111,12 +111,12 @@ Format: `help`
...
 Adds a contact.
 
-Format: `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GIT_USERNAME [t/TAGS]`
+Format: `addc n/CONTACTNAME p/PHONENUMBER e/EMAIL u/GITHUB_USERNAME [t/TAGS]`
 
 Examples:
 
 * `addc n/john p/12345678 e/john@nus.edu.sg u/john123 t/Schoolmate`
-* `addc n/mary p/87654321 e/mary@nus.edu.sg u/maryCS t/Teammate t/Classmate`
+* `addc n/mary p/87654321 e/mary@nus.edu.sg u/maryCS t/Teammmate t/Classmate`
```
Comment (snss231) [inline docs/UserGuide.md:119]: 2 ms

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java [hunk 4]
```diff
@@ -25,23 +25,25 @@
...
+            + "Usage: "
+            + COMMAND_WORD + " "
             + PREFIX_TASKNAME + "TASKNAME "
             + PREFIX_DATETIME + "DATETIME [, ENDDATETIME]"
-            + " [" + PREFIX_TAG + "TAG] "
-            + " [" + PREFIX_LINK + "LINK] "
-            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE]\n"
+            + " [" + PREFIX_TAG + "TAG" + "] "
+            + " [" + PREFIX_LINK + "LINK" + "] "
+            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE" + "] \n"
```
Comment (snss231) [inline src/main/java/seedu/address/logic/commands/AddTaskCommand.java:36]: Why these changes?

File: src/main/java/seedu/address/logic/commands/FindTaskCommand.java [hunk 5]
```diff
@@ -18,7 +18,9 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all tasks whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Usage: "
+            + COMMAND_WORD + " "
+            + "KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " apple orange pear";
```
Comment (snss231) [inline src/main/java/seedu/address/logic/commands/FindTaskCommand.java:24]: suggest replace " apple orange pear " with " lecture consultation"

**Review by snss231 — COMMENTED** [review]
made some suggestions, otherwise LGTM thanks Brian!

File: src/main/java/seedu/address/logic/commands/AddTaskCommand.java (hunk 4 above)
Comment (brian16600) [reply src/main/java/seedu/address/logic/commands/AddTaskCommand.java:36]: Yeah true, not really necessary. Changing them back

File: docs/UserGuide.md (hunk 3 above)
Comment (brian16600) [reply docs/UserGuide.md]: Edited

File: src/main/java/seedu/address/logic/commands/FindTaskCommand.java (hunk 5 above)
Comment (brian16600) [reply src/main/java/seedu/address/logic/commands/FindTaskCommand.java]: Okay changed
